bpfassets/libbpf/src/kepler.bpf.h

```diff
@@ -117,8 +117,10 @@ struct trace_event_raw_softirq {
 typedef struct process_metrics_t
 {
     u64 cgroup_id;
-    u64 pid;
-    u64 process_run_time;
+    u64 pid; // pid is the kernel space view of the thread id
+    u64 tgid; // tgid is the user space view of the process pid
+    u64 process_run_time_ms;
+    u64 process_run_time_ns;
```

Comment:
@rootfs where are you using the nanosecond?
Did this fix the problem that you had before?

Reply:
I found where you are using it.

If you are replacing the `ms` with `ns`, should we just remove the `ms`?

Again, did this fix the problem that you saw before?

Reply:
That is great, increment a counter in nanoseconds can potentially overflow.
It is better to keep using milliseconds.